<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomTripMenu</class>
 <widget class="QDialog" name="CustomTripMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>331</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>141</width>
     <height>226</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="MacDonalds_checkBox">
      <property name="text">
       <string>MacDonalds</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Chipotle_checkBox">
      <property name="text">
       <string>Chipotle</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Dominos_checkBox">
      <property name="text">
       <string>Domino's Pizza</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="KFC_checkBox">
      <property name="text">
       <string>KFC</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Subway_checkBox">
      <property name="text">
       <string>Subway</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="InNOut_checkBox">
      <property name="text">
       <string>In - N - Out Burger</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Wendys_checkBox">
      <property name="text">
       <string>Wendy's</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="JackInTheBox_checkBox">
      <property name="text">
       <string>Jack in the Box</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ElPolloLoco_checkBox">
      <property name="text">
       <string>El Pollo Loco</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="PapaJohns_checkBox">
      <property name="text">
       <string>Papa John's Pizza</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CustomTripMenu</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CustomTripMenu</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
